README.md

```diff
@@ -98,10 +97,10 @@ and Drop component to a lesson, then click the `EDIT` button.
 ![Edit view](https://raw.githubusercontent.com/edx-solutions/xblock-drag-and-drop-v2/c955a38dc3a1aaf609c586d293ce19b282e11ffd/doc/img/edit-view.png)
```

Comment:
@e-kolpakov I think the reason was we used to have a central repository to which we were copying xblock README files on each release. I don't think that is the case anymore and using relative links definitely makes much more sense.